README.md

```diff
@@ -108,3 +108,49 @@ More about how the upstream action works can be found [here](https://github.com/
 ### Add new components to Release Please config file
 
 In order for components to be released, they must be in the [release-please-config.json](./release-please-config.json) file. Always ensure new components are added to this file.
+
+Once the component is added, as part of using the latest version of an action, it's a good practice to update the README file of each shared workflow every time there is a new release.
```

Comment:
Along the lines of consistency, I would be a bit more direct and say that it is needed

```suggestion

`README` files for each component should have embedded versions updated every time there is a new release.
```